tests/linux/CMakeLists.txt

```diff
@@ -0,0 +1,19 @@
+target_sources(multipass_tests
+  PRIVATE
+    ${CMAKE_CURRENT_LIST_DIR}/../libvirt/mock_libvirt.cpp
+    ${CMAKE_CURRENT_LIST_DIR}/test_apparmored_process.cpp
+    ${CMAKE_CURRENT_LIST_DIR}/test_platform_linux.cpp
+    ${CMAKE_CURRENT_LIST_DIR}/test_qemuimg_process_spec.cpp
+    ${CMAKE_CURRENT_LIST_DIR}/test_snap_utils.cpp
```

Comment:
With `include` for libvirt this would point to `tests/`, not `tests/libvirt`.

Reply:
Right, I originally wrote the comment there but then skipped it because I imagined that would be the case. But then I saw it again here. For these regular cases, this looks intriguing.

Reply:
It's just a variable like any other, pointing to the directory where the `CMakeLists.txt` is. I'd leave it be.

Reply:
For future reference, the var is needed because, in older versions of cmake (before 3.13 I believe), relative paths would be resolved against the directory where the target was added (the parent dir, in this case), while in newer versions they would be resolved relatively to the current source directory (the same as `CMAKE_CURRENT_LIST_DIR` here). This way, it works on both sides. See https://cmake.org/cmake/help/latest/policy/CMP0076.html#policy:CMP0076